k8s/demo/busybox/localpv-device-pod.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright © 2019 The OpenEBS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Example for launching a busybox pod using localpv with device.
+# 
+# This example uses the default localpv with hostdevice 
+# storage class installed by OpenEBS. 
+#
+# Prior to running this YAML, verify that the storage class 
+# mentioned below is available. 
+#
```

Comment:
Can we add one more line saying that a device matching the storage requirement should be attached to node